### PR TITLE
Edit index.html

### PR DESCRIPTION
**Pull request by OksanaShulha**
Now our html file looks awesome according  with this example https://daily-dev-tips.com/posts/html5-starting-boilerplate-template/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
-    <body>
-        <!--[if lt IE 7]>       
-            <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
-        
-        <script src="" async defer></script>
-    </body>
+<head>
+  <meta charset="utf-8">
+  <title>Daily Dev Tips Boilerplate</title>
```
Comment (IrinaSing) [inline index.html]: It's nice but there is default content in tags.

<title> should be the name of the page that user sees in browser window. I suppose, for us it is Loruki.

File: index.html [hunk 2]
```diff
@@ -1,24 +1,16 @@
...
+  <meta name="description" content="Basic HTML boilerplate">
```
Comment (IrinaSing) [inline index.html]: And in Description content should be something about Loruki instead of "Basic HTML boilerplate"

File: index.html [hunk 3]
```diff
@@ -1,24 +1,16 @@
...
+  <meta name="author" content="Daily Dev Tips">
```
Comment (IrinaSing) [inline index.html]: Author is our team, in the template is still "Daily Dev Tips".

**Review by IrinaSing — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (IrinaSing) [reply index.html]: Maybe something like: "Official page of Loruki platform"

File: index.html (hunk 1 above)
Comment (OksanaShulha) [reply index.html]: You mean Home Page?

Comment (OksanaShulha) [reply index.html]: Ok I am going change for Loruki , anyway we can change it anytime during html-writing

Comment (IrinaSing) [reply index.html]: It supposed to be template for all the pages. If you open website in browser with this title:
<title>Daily Dev Tips Boilerplate</title>

You will see "Daily Dev Tips Boilerplate" written in the tab.

**Review by OksanaShulha — COMMENTED** [review]
Now it should be really awesome))

File: index.html (hunk 1 above)
Comment (IrinaSing) [reply index.html]: > Ok I am going change for Loruki , anyway we can change it anytime during html-writing

Yes, but I think we can just forget about it later 😅

**Review by IrinaSing — APPROVED** [review]
Thank you!

**Pull request merged**